xml/System.Net.WebSockets/WebSocket.xml

```diff
@@ -636,9 +626,7 @@
         <param name="buffer">References the application buffer that is the storage location for the received data.</param>
         <param name="cancellationToken">Propagates the notification that operations should be canceled.</param>
         <summary>Receives data from the <see cref="T:System.Net.WebSockets.WebSocket" /> connection asynchronously.</summary>
-        <returns>Returns <see cref="T:System.Threading.Tasks.Task`1" />.  
-  
- The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns a <see cref="T:System.Byte" /> array containing the received data.</returns>
+        <returns>The task object representing the asynchronous operation. The <see cref="P:System.Threading.Tasks.Task`1.Result" /> property on the task object returns a <see cref="T:System.Byte" /> array containing the received data.</returns>
```

Comment:
Possibly the detail about the Result property is better suited for remarks.

Reply:
I'm leaving these alone for now.